From dd06a90c6852e5b0efab3276f5a1b8e079abf659 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 16 Nov 2018 13:34:33 -0700
Subject: [PATCH 1/8] common: Expand labeling of sysfs_vibrator nodes using
 genfscon

* In particular, many Samsung kernels use pwm_*-based nodes for
  manipulation. As opposed to chasing all the variations, we should
  just assume that anything under the vibrator directory is a
  vibrator node.

Change-Id: I64824bb59c2103362e6875947840faf378b0dc48
---
 common/private/file_contexts  | 6 ------
 common/private/genfs_contexts | 2 ++
 2 files changed, 2 insertions(+), 6 deletions(-)

diff --git a/common/private/file_contexts b/common/private/file_contexts
index d48fa49..0e85dc1 100644
--- a/common/private/file_contexts
+++ b/common/private/file_contexts
@@ -25,9 +25,3 @@
 # Userinit
 /data/local/userinit\.sh                u:object_r:userinit_data_exec:s0
 /system/etc/init\.d/90userinit          u:object_r:userinit_exec:s0
-
-# Vibrator control
-/sys/devices/virtual/timed_output/vibrator/vtg_level u:object_r:sysfs_vibrator:s0
-/sys/devices/virtual/timed_output/vibrator/vtg_min u:object_r:sysfs_vibrator:s0
-/sys/devices/virtual/timed_output/vibrator/vtg_max u:object_r:sysfs_vibrator:s0
-
diff --git a/common/private/genfs_contexts b/common/private/genfs_contexts
index 93810ab..97887b9 100644
--- a/common/private/genfs_contexts
+++ b/common/private/genfs_contexts
@@ -1,2 +1,4 @@
 genfscon fuseblk / u:object_r:vfat:s0
 genfscon sdfat / u:object_r:exfat:s0
+
+genfscon sysfs /devices/virtual/timed_output/vibrator u:object_r:sysfs_vibrator:s0
-- 
2.17.1

